website/docs/content/writing-style/index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Writing Style
+caption: How HashiCorp structures writing, in a clear concise manner.
+description: How HashiCorp structures writing, in a clear concise manner.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
```

Comment:
Just noting this needs to be updated. I know you were soliciting feedback from folks but just to make sure it isn't forgotten about.